0_RootFS/llvm_common.jl

```diff
@@ -26,7 +27,7 @@ function llvm_script(;version = v"8.0.1", llvm_build_type = "Release", kwargs...
     LLVM_BUILD_TYPE=$(llvm_build_type)
     """ *
     raw"""
-    apk add build-base python-dev linux-headers musl-dev zlib-dev
+    apk add build-base python3 python3-dev linux-headers musl-dev zlib-dev
```

Comment:
I need to install `python2-dev` or `python3-dev` as `python-dev` no longer exists for Alpine linux:
```
[03:19:35]  ---> apk add build-base python-dev linux-headers musl-dev zlib-dev
[03:19:35] fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/x86_64/APKINDEX.tar.gz
[03:19:36] ERROR: unable to select packages:
[03:19:36]   python-dev (no such package):
[03:19:36]     required by: world[python-dev]
```
We probably don't need to install `python3` but it seemed weird as Python 2 seems to be the default installed